### ISSUE TITLE
Shairport hardcoded to headphones

### ISSUE THREAD

**Issue by dadr** [project-owner/Peppy#47]
For Kandinsky-official...   

I tried the "AirPlay" feature, and had to make changes to /etc/shairport-sync.conf    at about line 63 or so, there are two lines that specify "headphones".   After changing those to "default"  the module worked with the HiFiBerry output.

Also, a nit.  When a new song is played and does not have album art, Peppy continues to show the previous song's album art.

**Comment by project-owner** [comment]
You are right, I was always planning to make templates for ALSA, Bluetooth, Shairport, mpd and... always postpone that task. I should at least add that info to the wiki pages.

Regarding the album art, when does it happen? Which mode? 
Thank you!

**Comment by dadr** [comment]
The album art is when using the shairport module/mode.   The typical function on the Peppy UI is to show the album art of the current song.  (You can also change the volume level and skip back or to next.)   If a song is playing with album art, and the next song then plays without album art, then the Peppy UI continues to show the art from the previous song.  This can also happen if you are playing a song and skip back to a previous song that doesn't have album art.  When this occurs on an iPhone, it shows a large musical note in place of the missing album art.   I was expecting the see the picture of a CD.  That seems to be the similar graphic used in Peppy.  For example, when you start AirPlay, you see the CD before any other album art is shown.

**Comment by project-owner** [comment]
Got it. I'll fix that in the next release.
